docs/troubleshooting-guide/index.md

```diff
@@ -1,38 +1,96 @@
 
-- [Can't save questions or dashboards, or getting a blank page](./proxies.html)
+### Setup and administration
 
-- [LDAP](./ldap.html)
+- [Running the Metabase JAR][running].
 
-- [Email](./email.html)
+- [Running Metabase on Docker][docker].
 
-- [I think I found a bug](./bugs.html)
+- [Connecting to data warehouses with Metabase][datawarehouse].
 
-## Helpful tidbits
+- [Loading an exported application database][loadh2].
 
-### Accessing the Metabase server logs
+- [Setting up LDAP][ldap].
 
-You can always look for the logs that Metabase leaves on your server's file system (or wherever else you collect logs). If you're logged into Metabase with an admin account, you can also access the logs from the **Logs** tab in the **Troubleshooting** section of the **Admin Panel**. To get to the Admin Panel, click the **gear icon** in the top-right of Metabase.
+- [Setting up Email][email].
 
-### Checking for Javascript console errors
+- [Using the default H2 application database][appdb].
 
-Metabase will send debugging information and errors to your browser's developer console. To open the developer console, follow the instructions for your web browser of choice:
+- [Handling timezones][timezones].
 
-- [Chrome](https://developers.google.com/web/tools/chrome-devtools/open#console)
-- [Firefox](https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console)
-- [Safari](https://support.apple.com/guide/safari-developer/develop-menu-dev39df999c1/mac)
-- [Edge](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium)
+- [Manage data sandboxing][sandbox]
+
+## Think you found a bug?
+
+Let us know by [filing a bug report][bugs].
+
+## More troubleshooting resources
+
+Here are some other resources to help you solve your problem or find a workaround.
+
+### Metabase server and console logs
+
+Metabase will log errors, both on the server and in the browser console, depending on where the error occurs, which can help you track down an issue. Administrators will have access to the server logs, and everyone with a browser can open the developer tools to see the console logs.
+
+**Accessing the Metabase server logs**: You can look for the logs that Metabase leaves on the server's file system (or wherever else you collect logs). If you're logged into Metabase with an Admin account, you can also access the logs by clicking on the **gears icon** in the top right of the main nav, selecting **Admin**, clicking on the **Troubleshooting** tab, then viewing the **Logs** tab.
+
+**Checking for Javascript console errors:** Metabase will send debugging information and errors to your browser's developer console. To open the developer console, follow the instructions for web browser:
+
+- [Chrome][chrome]
+- [Firefox][firefox]
+- [Safari][safari]
+- [Edge][edge]
+
+### Metabase tutorials 
+
+For tutorials that walk you through how to use Metabase features, check out [Learn Metabase][learn].
```

Comment:
I'm not a big fan of single-line sections, but I think it's appropriate in this case.